client/pages/signup.jsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import { ApolloConsumer } from "react-apollo";
+import gql from "graphql-tag";
+
+import HomeLayout from "../components/layouts/HomeLayout";
```

Comment:
It's better if you can change the filename and variable name to `Layout` since its use is not limited to the landing page.

Reply:
Done!